corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -701,7 +701,7 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
                 $.ajax({
```

Comment:
^^ for the icons above, we generally use fontawesome: http://fontawesome.io/icons/. realize those are the same icons as mobile, but they don't really match HQ. @biyeun @orangejenny any thoughts?